systemd-swap.service

```diff
@@ -1,9 +1,9 @@
 [Unit]
-Description=Manage swap spaces on zram, files and partitions.
+Description=Manage swap spaces on zram, files, and partitions.
```

Comment:
Afaik there shouldn't be a comma here x2

Reply:
It's called the [Oxford comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/). It's entirely optional to use, but increases readability IMO. Anyway, I'm going to move those unrelated changes to different PRs.